sdk/tables/data-tables/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+- Updated our `@azure/core-tracing` dependency to the latest version (1.0.0-preview.14)
```

Comment:
Question for the team - does this belong in the breaking changes section? Removing @opentelemetry/api is technically breaking right but since it's all in preview the water is a little muddy there. What do you think?  cc @xirzec ?

If it should go under breaking changes I'll move the changelog entry here and in eventhubs

Reply:
I think it's fine under Other for now. Really looking forward to getting this out of preview.